screens/MenteesListView.js

```diff
@@ -60,19 +43,12 @@ class MenteeListView extends Component {
   render() {
     return (
       <View style={styles.contentWrap}>
-        <View style={styles.floatingView}>
-          <ActivityIndicator
-            animating={this.state.loading}
-            size="large"
-            color="#0000ff"
-          />
-        </View>
         <ScrollView
           refreshControl={(
             <RefreshControl
               refreshing={this.props.refreshingContacts}
               onRefresh={() => {
-                this.props.dispatch(constructContactItemsWithToken(JSON.parse(this.state.hmToken)));
+                this.props.dispatch(constructContactItemsWithToken(this.props.user.hmToken));
```

Comment:
What happens for the refresh controller the very first time the app is loaded? Does onRefresh get triggered?

Currently there's a double load of the mentee list page the first time I run the app with nothing in the redux state (reproducible by clearing the user and running again). 

By "double load" I mean that I see the list get generated once, cleared, then generated a second time rapidly.

Reply:
added conditions to not let the "double load" situation happen